### ISSUE TITLE
Implement RenderPassAttachmentsList<Arc<T>> on render passes with only one attachment

### ISSUE THREAD

**Issue by tomaka** [tomaka/vulkano#106]
So that instead of: 

``` rust
Framebuffer::new(&device, [1024, 768, 1], render_pass::AList { color: &img })
```

You can write:

``` rust
Framebuffer::new(&device, [1024, 768, 1], &img)
```

**Comment by tomaka** [comment]
No longer relevant after #486 which removed the trait.
